README.md

```diff
@@ -1,15 +1,23 @@
 # Kueue
 
-Kubernetes-native Job Queueing
+Kueue is a set of APIs and controller for job queueing. It is a job-level manager that decides when 
+a job should start (as in pods can be created) and when it should stop (as in active pods should be 
+deleted). The main design principle for Kueue is to avoid duplicating existing functionality: autoscaling, 
+pod-to-node scheduling, job lifecycle management and advanced admission control are the responsibility of 
+existing k8s components, namely cluster-autoscaler, kube-scheduler, kube-controller-manager and Gatekeeper, 
```

Comment:
let's not imply that gatekeeper is a k8s component.

Reply:
done.